metta/reduct/rules.metta

```diff
@@ -0,0 +1,59 @@
+(: chain-reduction-rule (-> Atom))
+(: recurse-reduction-rule (-> Atom))
+(: commutative-reduction-rule (-> Atom))
+
+(= (chain-reduction-rule)
+   (: ChainReduction
+      (-> (examplar $a)
+          (reduced $a $b)
+          (examplar $b)))
+)
+
+(= (commutative-reduction-rule)
+   (: CommutativeReduction
+      (-> (reduced (andd $b $c) $r)
+            (is-commutative andd)
+            (reduced (andd $c $b) $r)))
+)
+
+(= (commutative-reduction-rule)
+   (: CommutativeReduction
+      (-> (reduced (orr $b $c) $r)
+            (is-commutative orr)
+            (reduced (orr $c $b) $r)))
+)
+
+(= (recurse-reduction-rule)
+   (: RecurseReduction
+      (-> (examplar (orr $b $c))
+            (reduced $c $rc)
+            (reduced (orr $b $c) (orr $b $rc))))
```

Comment:
In addition I think you should have two premises
1. `$c` reduces to `$rc`
2. `$b` reduces to `$rb`
That way you can also get rid of the rule below.